framework/Models/PostProcessors/BasicStatistics.py

```diff
@@ -290,10 +301,11 @@ def initialize(self, runInfo, inputs, initDict):
     metaKeys = inputMetaKeys + outputMetaKeys
     self.addMetaKeys(metaKeys,metaParams)
 
-  def _handleInput(self, paramInput):
+  def _handleInput(self, paramInput, childVals=[]):
```

Comment:
having an empty list as an optional argument in Python has unexpected consequences as a mutable object, see e.g. https://docs.python-guide.org/writing/gotchas/. Better off setting the default to `None` then checking `if None: val = []` in the beginning of the method.

Reply:
Changed default to None in commit 199977a